client/components/inline-help-happychat/style.scss

```diff
@@ -0,0 +1,34 @@
+.button.inline-help-happychat {
+	z-index: z-index( 'root', '.floating-help' );
+	margin: 8px;
+	line-height: 0;
+	padding: 2px;
+	border-radius: 100%;
+	background: $blue-wordpress;
+	box-shadow: 0 2px 6px rgba( 0, 0, 0, 0.15 );
+	border: 1px solid darken( $blue-wordpress, 5 );
+
+	.gridicon {
+		fill: $white;
+		margin: 2px;
+		top: 0 !important;
+		height: 24px;
+		width: 24px;
+	}
+
+	// We have to over-ride .button.is-borderless styles here.
+	&.button.is-borderless {
```

Comment:
I'm not sure what happens when this is transpiled - won't the `&` take us back up to the next level and give us a CSS selector `.button.inline-help-happychat.button.is-borderless`?

Reply:
Good spot thanks! This is a bit of copy-pasta! I'll get rid of it :)